### DISCUSSION TITLE
Issues using both Flow Deck and AI Deck underneath the CrazyFlie

### DISCUSSION THREAD

**Discussion by stmbl0** [bitcraze/discussions#1718] [Q&A]
We are working on a project that requires the camera of the AI Deck to be pointed downwards. We modified the AI Deck (by removing the original camera mount and replacing it) and mounted it at the bottom of the drone.
Everything was fine and worked until we added the Flow Deck, which we will also need. Once the flow deck was added, no messages from the GAP8 processor showed up in the console and it was not reachable through the WiFI-Network. When the flow deck was disconnected or the AI Deck was moved to the top of the drone, everything worked as expected (however the camera was blocked by the antenna).

Is there any way to mount both decks below the drone and have them function as expected?
Thanks in advance!

**Comment by stmbl0** [comment]
After more observations, the issue mostly comes up after rebooting the drone a few times with both decks attached. Then the GAP8 does not seem to send any messages either after a OW Timeout or just basically out of nowhere.

Crazyflie still detects both decks, as indicated by `DECK_CORE: 2 deck(s) found` and there are still messages sent by the ESP32 over CPX, just nothing from the GAP8. The Flow Deck behaves and works exactly as expected.

Another thing i've discovered is that it works way more consistently when the drone's battery is fully charged. I've also seen WiFi disconnects when the drone took off (-> when it the motors drew more power)

**Reply by gemenerik** [reply]
You're absolutely right—the physical placement of the decks, whether above or below the main PCB, shouldn't affect functionality.

We've observed some initialization issues with the AI-deck, which may also be influenced by the presence of other decks.

If you have the opportunity, please try the resistor workaround mentioned in https://github.com/bitcraze/aideck-gap8-examples/issues/105. Adding a 4.5 kΩ resistor between VCOM and ground can help the system to shut off cleanly. Please let me know if this resolves the issue.
